_posts/2019-02-12-funding-eslint-future.md

```diff
@@ -0,0 +1,66 @@
+* **Automatic Invoicing.** For corporate sponsors, Open Collective automatically generates and sends invoices for tracking purposes.
+* **Open Participation.** Anyone can request reimbursement for funds spent helping the ESLint project and ESLint can pay out to anyone.
+
+## How will the money be spent?
+
+The ESLint team has a lot of ideas about how to proceed going forward. Depending on the amount we raise, here are some of the things we're planning:
+
+* **Paying team members for maintenance and development.** All ESLint team members are still contributing in their spare time. Having the option to work part-time or full-time on ESLint, and be compensated for it, would allow for faster and more consistent development.
+* **Rewarding contributions from the community.** We'd like to reward everyone who contributes to ESLint, whether it's a one-time contribution or ongoing contributions. We are still figuring out the details, but rewarding people with everything from stickers and t-shirts to cash payouts for significant contributions are things we are thinking about right now.
+* **Improve documentation.** ESLint's documentation hasn't had a significant overhaul since the project was first released. We feel there are a lot of ways to improve the documentation, including potentially contracting an information architect or technical writer to improve our documentation.
+* **Better communication and support.** By paying team members part-time or full-time, that also opens up the possibility of sending team members to meetups, companies, and conferences to interact with the community and better understand the problems developers are facing.
+* **Developing a Roadmap.** For a while, the ESLint team has mostly worked on a one-off basis in terms of which features to implement. Without knowing who would be able to spend what amount of time on the project, it was difficult to do long-term planning and develop a roadmap for the future. This is something we will finally be able to do when we are a self-sustaining project.
+
+## Our Goal: $20,000 per month
+
+In order to achieve everything we have in mind, we believe that we will need $20,000 per month. With this amount, we can sustain ESLint for the foreseeable future.
+
+## Introducing our first sponsors
+
+<p class="text-center">
+    <a href="https://facebook.com"><img src="/img/logos/facebook.png" alt="Facebook" height="96"></a>
+    <a href="https://airbnb.com"><img src="/img/logos/airbnb.png" alt="Airbnb" height="96"></a><br>
+    <a href="https://frontendmasters.com"><img src="https://static.frontendmasters.com/assets/fm/js/frontendmasters.0e71088726.svg" alt="Frontend Masters" height="32"></a>
+</p>
+
+We're also excited to announce our first sponsors. [Facebook](https://facebook.com) and [Airbnb](https://airbnb.com) have both pledged $1,000 per month to support ESLint.
+
+Additionally, [Frontend Masters](https://frontendmasters.com) has pledged $200 per month to support ESLint.
```

Comment:
I need to double check on this. I thought this was a monthly donation, but it looks like it was a one-time donation. I'm going to try to reach Marc to make sure that was intended so I can reword this.

Reply:
Okay, this was a one-time donation to get us started, I'll update the post.